### DISCUSSION TITLE
Reboot info for update-lxc

### DISCUSSION THREAD

**Discussion by MajorTomDE** [tteck/Proxmox#1899] [Ideas]
Hey,
it would be great if update-lxc shows an info when a reboot is required.
e.g. as an second option

1 = Classic Update
2 = Check Update Status > Show count of available updates, reboot required, disk status

Result should be a list that shows the infos for all lxc machines.

**Comment by tteck** [comment]
Something like this? 

![Screenshot 2023-10-08 9 56 23 AM](https://github.com/tteck/Proxmox/assets/21342670/b820314f-d0dc-40a6-aac7-0356e0f44982)

**Reply by MajorTomDE** [reply]
Yes, great :-) Maybe in addition the container name to the ID. Then it’s perfect.

**Reply by tteck** [reply]
![Screenshot 2023-10-08 12 08 22 PM](https://github.com/tteck/Proxmox/assets/21342670/21fc4a6b-0f3e-42bb-817d-950d99189f22)
